server/etc/rc.d/init.d/pulp_workers

```diff
@@ -59,6 +60,10 @@ if [ ! -z "$CELERY_APP" ]; then
     CELERY_APP_ARG="--app=$CELERY_APP"
 fi
 
+# Sets --maxrequestsperchild argument for CELERY_BIN
+PULP_MAX_TASKS_PER_CHILD=${PULP_MAX_TASKS_PER_CHILD:-$DEFAULT_PULP_MAX_TASKS_PER_CHILD}
+CELERY_MAX_TASKS_PER_CHILD_ARG="--maxrequestsperchild=$PULP_MAX_TASKS_PER_CHILD"
```

Comment:
This will include `maxrequestsperchild` on all workers that are started. That is OK if a value of 0 leaves the feature disabled. [The docs](http://celery.readthedocs.io/en/latest/configuration.html#std:setting-CELERYD_MAX_TASKS_PER_CHILD) don't indicate what a value of 0 would do. Can you confirm that a default of 0 will disable the feature?